lib/components/term.js

```diff
@@ -20,11 +20,27 @@ const CURSOR_STYLES = {
   BLOCK: 'block'
 };
 
+const isWebgl2Supported = (() => {
+  let isSupported = window.WebGL2RenderingContext ? undefined : false;
+  return () => {
+    if (isSupported === undefined) {
+      const canvas = document.createElement('canvas');
+      const gl = canvas.getContext('webgl2', {depth: false, antialias: false});
+      isSupported = gl instanceof window.WebGL2RenderingContext;
+      if (!isSupported) {
+        // eslint-disable-next-line no-console
+        console.warn(`WebGL2 is not supported on your machine. Falling back to canvas-based rendering.`);
```

Comment:
I would move this warning in `getTermOptions` where fallback occurs. This is juste a test here.

We can warn if webGL is disabled due to a non-opaque backgroundColor too.

Reply:
Done!

Reply:
Sorry, linting issues on push. One sec

Reply:
Ok, *now* it's done. I need to set up eslint integration in vim.